.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think you'll want to add `.pytest_cache/` to your `.gitignore` and delete this directory & the files in it.

Reply:
Good catch!